docs/docs/launch-manual/sections/using-pulsar/sections/pulsar-packages.md

```diff
@@ -119,16 +121,29 @@ $ ppm search linter
 > └── linter-rust Lint Rust-files, using rustc and/or cargo (132550 downloads, 91 stars)
 ```
 
-You can use `ppm view` to see more information about a specific package.
+You can use `pulsar -p view` to see more information about a specific package.
 
 ```sh
-$ ppm view linter
+$ pulsar -p view linter
 > linter
 > ├── 3.4.0
 > ├── https://github.com/steelbrain/linter
 > ├── A Base Linter with Cow Powers
 > ├── 9863242 downloads
 > └── 4757 stars
 >
-> Run `ppm install linter` to install this package.
+> Run `pulsar -p install linter` to install this package.
 ```
+
+#### Using ppm to install from other sources
+
+By default ppm will be using the [pulsar-edit.dev package site](https://web.pulsar-edit.dev/).
```

Comment:
```suggestion
By default ppm will be using the [Pulsar Package Repository](https://web.pulsar-edit.dev/).
```

What do you think about using this for our official name for the backend?

Reply:
More than happy with it, I don't think we have had an official name up until now.